server/index.js

```diff
@@ -1,10 +1,15 @@
-var express = require('express');
-var app = express();
+const express = require('express');
+const db = require('../database/index.js');
+const app = express();
 
-let port = 3004;
+const port = 3004;
 
 app.use(express.static(__dirname + '/../client/dist'));
 
 app.listen(3004, function() {
   console.log(`listening on port ${port}`)
-})
\ No newline at end of file
+});
+
+app.get('/getRooms', (req, res) => {
+  db.getAllRecords(res.send.bind(res));
```

Comment:
If you have time, try changing them to promises. There should be a way you don't need to bind "this".